src/null-markdown-loader.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-env node */
+
```

Comment:
A comment here explaining why this file exists would help future maintenance.

Reply:
Sean 👀